src/hdf5_back.cc

```diff
@@ -1541,6 +1541,33 @@ void Hdf5Back::CreateTable(Datum* d) {
   schemas_[d->title()] = dbtypes;
 }
 
+std::map<std::string, DbTypes> Hdf5Back::ColumnTypes(std::string table) {
```

Comment:
It has been rebased. 

```bash
$ git pull upstream develop 
From github.com:cyclus/cyclus
 * branch            develop    -> FETCH_HEAD
Already up-to-date.
scopatz@ares ~/cyclus develop $ git checkout rr
Switched to branch 'rr'
Your branch is up-to-date with 'origin/rr'.
scopatz@ares ~/cyclus rr $ git rebase develop 
Current branch rr is up to date.
```

Not sure why this is here. Maybe @rwcarlsen rebased and force pushed? It is on develop. https://github.com/cyclus/cyclus/commit/8c72577f0fb29c03668a2ee1513270fa50d3dea5